README.md

```diff
@@ -119,32 +119,66 @@ simultaneously.
   <summary>Click to expand</summary>
 
   ```erlang
-  Conn = ?config(conn, Opts),
-  MigrationCall =
-    pure_migrations:migrate(
-      "scripts/folder/path",
-      fun(F) -> epgsql:with_transaction(Conn, fun(_) -> F() end) end,
-      fun(Q) ->
-        case epgsql:squery(Conn, Q) of
-          {ok, [
-            {column, <<"version">>, _, _, _, _, _},
-            {column, <<"filename">>, _, _, _, _, _}], Data} ->
-              [{list_to_integer(binary_to_list(BinV)), binary_to_list(BinF)} || {BinV, BinF} <- Data];
-          {ok, [{column, <<"max">>, _, _, _, _, _}], [{null}]} -> -1;
-          {ok, [{column, <<"max">>, _, _, _, _, _}], [{N}]} ->
-            list_to_integer(binary_to_list(N));
-          [{ok, _, _}, {ok, _}] -> ok;
-          {ok, _, _} -> ok;
-          {ok, _} -> ok;
-          Default -> Default
-        end
-      end),
-  ...
-  %% more preparation steps if needed
-  ...
-  %% migration call
-  ok = MigrationCall(),
+migrate() ->
+    Conf = ?config(super_account),
+    Path = ?env(scripts_path),
+    {ok, Conn} = epgsql:connect(Conf),
+    MigrationCall =
+      pure_migrations:migrate(
+        Path,
+        fun(F) -> epgsql:with_transaction(Conn, fun(_) -> F() end) end,
+        fun(Q) ->
+          case epgsql:squery(Conn, Q) of
+            {ok, [{column, <<"version">>, _, _, _, _, _, _, _},
+                   {column, <<"filename">>, _, _, _, _, _, _, _}], []} ->
+                    [];
+            {ok, [{column, <<"version">>, _, _, _, _, _, _, _},
+                   {column, <<"filename">>, _, _, _, _, _, _, _}], Data} ->
+                [{list_to_integer(binary_to_list(BinV)), binary_to_list(BinF)} || {BinV, BinF} <- Data];
+            {ok, [{column, <<"max">>, _, _, _, _, _, _, _}], [{null}]} ->
+                % It has to be -1 or it will get an error during initialization
+                -1;
```

Comment:
@leeyisoft had to change this to 0, otherwise I get a `{badmatch,{error,unexpected_version,{expected,0,supplied,1}}}`